spec/models/manageiq/providers/amazon/aws_refresher_spec_common.rb

```diff
@@ -48,7 +48,7 @@ def expected_table_counts
       :floating_ip                   => 12,
       :guest_device                  => 0,
       :hardware                      => 46,
-      :miq_queue                     => 50,
```

Comment:
ok. can you dig some deeper? I dont think we should merge it without knowing why it changed... after all these numbers are specs!